spark-3.5-scala-2.13.yaml

```diff
@@ -0,0 +1,145 @@
+package:
+  name: spark-3.5-scala-2.13
+  version: 3.5.3
+  epoch: 0
+  description: Unified engine for large-scale data analytics
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - bash
+      - busybox
+      - openjdk-17-default-jvm
+      - posix-libc-utils
+      - procps
+      - tini
+      - tini-compat
+
+environment:
+  contents:
+    packages:
+      - bash
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - glibc-iconv
+      - glibc-locale-en
+      - grep
+      - maven
+      - openjdk-17-default-jdk
+      - perl-utils
+      - procps
+      - py3.11-pip
+      - python-3.11
+      - wolfi-base
+      - wolfi-baselayout
+      - yaml-dev
+  environment:
+    LANG: en_US.UTF-8
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/apache/spark
+      tag: v${{package.version}}
+      expected-commit: 32232e9ed33bb16b93ad58cfde8b82e0f07c0970
+
+  - runs: |
+      export JAVA_HOME=/usr/lib/jvm/java-17-openjdk
+      export MAVEN_OPTS="-Xss64m -Xmx2g -XX:ReservedCodeCacheSize=1g"
```

Comment:
These can go in the environment, since you install `-default-jdk` though, setting JAVA_HOME shouldn't be necessary

Reply:
Moved to environment